src/swish/stream.ss

```diff
@@ -188,15 +204,26 @@
               x)
             eos))))
 
-  (define (->stream x)
+  (define (->stream x ht?)
```

Comment:
Consider having `s/>` convert hashtables to streams so that we don't need the ht? argument.

Reply:
I was trying to avoid breaking code like `(s/> ht s/cells)`. I've changed `s/>` to do the conversion and `s/cells` to be a no-op if the input isn't a hashtable. There probably isn't code in the wild that kicks off the stream with a hashtable and the first transformer expects a hashtable.